gamestonk_terminal/cryptocurrency/cryptocurrency_helpers.py

```diff
@@ -972,6 +1150,38 @@ def plot_chart(
 
             console.print("")
 
+    if source == "yf":
+        symbol_yf = coin_map_df["YahooFinance"]
+
+        df = yf.download(
+            symbol_yf,
+            end=datetime.now(),
+            start=datetime.now() - timedelta(days=days),
+            progress=False,
+            interval=interval,
+        ).sort_index(ascending=False)[
+            [
+                "Open",
+                "High",
+                "Low",
+                "Close",
+                "Volume",
+            ]
+        ]
```

Comment:
We have `plot_chart` exposed to the notebooks api. and don't have `plot_candles` exposed. Let's expose `plot_candles` so that we can have the chart in the notebook with the data that we previously loaded